src/store/getters/state-getters.ts

```diff
@@ -42,9 +42,9 @@ export const isPremiumAccount = (state: any): boolean => {
   return (state.stateModel.accountInformation.accountType === AccountTypes.PREMIUM)
 }
 
-/** Whether the current filing is future effective. */
-export const isFutureEffective = (state: any): boolean => {
```

Comment:
`isFutureEffective` was not used, and we needed `getEffectiveDate`.

Reply:
Good clean up!